homeassistant/components/bmw_connected_drive/sensor.py

```diff
@@ -53,6 +55,8 @@ class BMWSensorEntityDescription(SensorEntityDescription):
     "charging_status": BMWSensorEntityDescription(
         key="charging_status",
         icon="mdi:battery-charging",
+        device_class="battery",
```

Comment:
Please import the sensor device class constants and use here.
```suggestion
        device_class=DEVICE_CLASS_BATTERY,
```